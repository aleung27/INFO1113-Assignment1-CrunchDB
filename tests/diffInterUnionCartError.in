SET a 0
SET b 4 2 4
DIFF a c
UNION a c
INTER a c
CARTPROD a c
POP a
DIFF a b
UNION a b
INTER a b
CARTPROD a b
POP b
POP b
POP b
DIFF a b
UNION a b
INTER a b
CARTPROD a b
BYE
