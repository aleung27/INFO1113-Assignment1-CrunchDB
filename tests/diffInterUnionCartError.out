> ok

> ok

> no such key

> no such key

> no such key

> no such key

> 0

> [2 4]

> [2 4]

> []

> [ [] ]

> 4

> 2

> 4

> []

> []

> []

> [ [] ]

> bye
